src/machine/CoffeeMachine.java

```diff
@@ -0,0 +1,92 @@
+package machine;
+
+class CoffeeMachine {
+
+    private Supplies supplies;
+
+    // It's ok, that the type of the price is int for this task
+    private int money;
+
+    private CoffeeMachineState currentState;
+
+    CoffeeMachineState getCurrentState() {
+        return currentState;
+    }
+
+    CoffeeMachine() {
+        supplies = new Supplies(400, 540, 120, 9);
+        this.money = 550;
+        this.currentState = CoffeeMachineState.CHOOSING_AN_ACTION;
+    }
+
+    private void displayContent() {
+        System.out.println("\nThe coffee machine has:");
+        System.out.printf("%d ml of water\n", supplies.getAmountOfWater());
+        System.out.printf("%d ml of milk\n", supplies.getAmountOfMilk());
+        System.out.printf("%d g of coffee beans\n", supplies.getAmountOfCoffeeBeans());
+        System.out.printf("%d disposable cups\n", supplies.getAmountOfDisposableCups());
+        System.out.printf("$%d of money\n", money);
+    }
+
+    private void prepareCoffee(int chosenOption) {
+        final var coffee = Coffee.fromOption(chosenOption);
+        if (coffee != null) {
```

Comment:
A bit shorter way: `if (coffee == null || !isEnoughIngredients(coffee)) return;`

Also this way we don't need to add extra 4 spaces for the following lines